cmd/api/src/model/appcfg/flag.go

```diff
@@ -109,6 +110,13 @@ func AvailableFlags() FeatureFlagSet {
 			Enabled:       false,
 			UserUpdatable: false,
 		},
+		FeatureDarkMode: {
+			Key:           FeatureDarkMode,
+			Name:          "Dark Mode",
+			Description:   "Reveals a toggle in the settings menu that can be used to toggle dark mode on and off",
```

Comment:
nit: using "toggle" twice doesn't read well to me. suggest "Allows users to enable or disable dark mode via a toggle in the settings menu"

Reply:
Agreed! Updating this.